tests/integration-tests/chaos-experiments/helpers/requestCreator.js

```diff
@@ -16,13 +16,146 @@ module.exports = {
 
 async function init() {
     try {
-        app = await expressApp();
+        nockK8sChaosExperimentSupportedKinds();
+        const appInitUtils = require('../../testUtils');
```

Comment:
I gotta feeling that maybe this can be reverted now

Reply:
removed the in place require

Reply:
removed the inplace require